Integrar excepciones y DTO genéricos en MedioPago

marquuus721 opened the pull request.

marquuus721 committed:
Integrar excepciones y DTO genéricos en MedioPago

ahumadamob commented:
No es correcta la ubicación de los archivos, deberías haber comenzado desde archivos existentes.

marquuus721 commented:
Si profe, anteriormente desarrollaba mi proyecto (rama develop) por la ubicación springboot-smedico2/src/main/java/com/imb2025/smedico . Pero en el TP04 subi mi desarrollo de entidad MedioPago reubicado en src/main/java/com/imb2025/smedico con el resto de entidades desarrolladas, ya que antes habia quedado separado por un error que cometi en anteriores practicos: 
habia colocado una carpeta llamada springboot-smedico-2 dentro del proyecto springboot-smedico seguida de el resto de carpetas herencia ....src/main/java/com/imb2025/smedico .
Ahora desarrollo mi entidad con el resto en src/main/java/com/imb2025/smedico.

marquuus721 committed:
Correcion Error - Integrar excepciones y DTO genéricos en MedioPago

marquuus721 commented:
Ya hice la corrección de mi ubicación.

marquuus721 committed:
Delete springboot-smedico/src/main/java/com/imb2025/smedico directory

La anterior ubicacion que agregue a mi rama tp04-quispe-marcos ya ha sido eliminada de la misma

marquuus721 commented:
Mi rama tp04-quispe-marcos quedo con la misma conformación de carpetas de la rama develop

marquuus721 committed:
Merge branch 'develop' into tp04-quispe-marcos

ahumadamob commented on src/main/java/com/imb2025/smedico/dto/MedioPagoRequestDto.java:
```diff
@@ -6,11 +6,10 @@
+
+
+  private String nombre;
+  private String tipo;
```
Te han faltado ahora las validaciones

marquuus721 replied:
Si profe, pero no coloque validaciones porque corresponden al tp05. Las diferencias que me figuraron entre la rama develop con esta rama tp04-quispe-marcos es porque develop ya habia sido actualizada con lo desarrollado en la rama tp05-quispe-marcos. Y entiendo que fusionar esta rama (tp04) con develop desactualizaria la develop.
